### DISCUSSION TITLE
[HELP] Finetuning Arabic Recognition model for Urdu language

### DISCUSSION THREAD

**Discussion by shariqhameedca** [PaddlePaddle/PaddleOCR#10581] [Q&A]
Hi,

I'm working on finetuning the Arabic model for recognizing Urdu text. Currently, there are few issues I'm facing while using the Arabic model for Urdu. 
- In some cases, Urdu text is not detected by the model. (Probably, issue with detector model)
- Urdu and Arabic have different dictionaries. 
- Even if the model detects the text and the characters in the text are present in Urdu as well as Arabic, still the model suffers to recognize the text. (Not in most cases).
- Some letters are recognized incorrectly. For example, the letter "ق" is not recognized, the letter "خ" is recognized as the "ف".

So, I need a little guidance. Is the finetuning approach correct? I have checked out this repository https://colab.research.google.com/drive/1QO5ZHPFgdOujHRduv2Dd_YVkzRikcHX0?usp=sharing#scrollTo=jdcs5DJVt1ip.

It is finetuning on Thai language. I want to know if I have to play with labels or something because Urdu is an RTL language.

**Comment by hritikakolkar** [comment]
Hi shariq have you solved the issue. If yes, can you help me with some info

**Comment by haroon423** [comment]
I'm working on paddleocr model  recognizing english or Urdu text. Currently, there are few issues I'm facing while using the paddleocr model for Urdu or englsih but english text extract fine paddleocr but issues with urdu text its not extract.

In some cases, Urdu text is not detected by the model. (Probably, issue with detector model)
Urdu and Arabic have different dictionaries.
Even if the model detects the text and the characters in the text are present in Urdu as well as Arabic, still the model suffers to recognize the text. (Not in most cases).
Some letters are recognized incorrectly. For example, the letter "ق" is not recognized, the letter "خ" is recognized as the "ف".
